scripts/release/create-docs-prs.sh

```diff
@@ -25,11 +25,16 @@ main() {
     # Check that version env exists
     [ -n "$VERSION" ]
     brew_install_if_needed jq
-    brew_install_if_needed gh
 
     step "Update mapbox/maps-ios@publisher-staging"
     maps_ios_upload_docs
 
+    # if STAGING_ONLY is set to true, then exit
+
+    # Exit is should be staging only
```

Comment:
```suggestion
```
I recommend removing this as the above comment says the same

Reply:
Done